budget/web.py

```diff
@@ -119,13 +150,21 @@ def authenticate(project_id=None):
 
 @main.route("/")
 def home():
-    project_form = ProjectForm()
+    # If ADMIN_PASSWORD is empty we consider that we are not in admin mode
+    admin = bool(current_app.config['ADMIN_PASSWORD'])
     auth_form = AuthenticationForm()
+
+    if admin:
+        # If we are in admin mode we don't need to pass a project form to home
```

Comment:
The comment is a bit confusing, I thought we were in a different view than "home". Maybe state "We don't need to pass the project form to the template".